src/Sulu/Bundle/CoreBundle/DependencyInjection/Compiler/RemoveForeignContextServicesPass.php

```diff
@@ -31,8 +31,8 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        $taggedServices = $container->findTaggedServiceIds(self::SULU_CONTEXT_TAG);
 
```

Comment:
You didn't remove it, you moved it ;-) But now it's not nicely formatted anymore.